modules/ocf_desktop/files/stats/update.sh

```diff
@@ -1,15 +1,21 @@
 #!/bin/bash
-# imvirt spews dmesg errors on stderr
-# https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=842226
-[ "$(imvirt 2>/dev/null)" == "Physical" ] || exit 0 # only count hours on desktops
```

Comment:
Why is this removed? I think we still only want to count hours on _physical_ desktops, not on VMs set up like desktops (for example).

Reply:
good point, I did not consider that case.

Reply:
It's not especially relevant right now since all our desktop setups are physical, but there used to be a VM set up like a desktop (informally, but it had the same scripts and stuff that a desktop had)

Reply:
Maybe edit the comment to explain why there might be non-physical "desktops" then